client/flutter/lib/pages.dart

```diff
@@ -0,0 +1,5 @@
+export './home_page.dart';
+
+export './pages/protect_yourself.dart';
+export './pages/travel_advice.dart';
+export './pages/who_myth_busters.dart';
```

Comment:
Is this necessary?

Reply:
It's to make the code more organized.
Since the idea is to use named routes, the other pages would have to be imported in the Router file, and that can make the file quite heavy. This way they pages are all organized and we only have to import this file in the Router file.